[김성주] sprint2

626-ju opened the pull request:
## 요구사항

### 기본
Netlify에 파일 배포가 아닌 포크한 Github 레포지토리로 연결합니다. 
[네트리파이 이동](https://panda-sprint2.netlify.app/)

- [x] 아래로 스크롤 해도 상단 네비게이션 바(Global Navigation Bar)가 최상단에 고정됩니다.
- [x] "판다마켓" 클릭 시 루트 페이지("/")로 이동합니다.(새로고침)
- [x] 로그인 페이지, 회원가입 페이지 모두 로고 위 상단 여백이 동일합니다.
- [x]  SNS 아이콘들은 클릭시 각각 실제 서비스 홈페이지로 이동합니다.
- [x] 로그인 페이지 "회원가입"버튼 클릭 시 "/signup" 페이지로 이동합니다.
- [x] 회원가입 페이지  "로그인"버튼 클릭 시 "/login" 페이지로 이동합니다.

### 심화
- [x] palette에 있는 color값들을 css 변수로 등록하고 사용해 주세요.
- [x] 비밀번호 input 요소 위에 비밀번호를 확인할 수 있는 아이콘을 추가해 주세요.

## 주요 변경사항
-
## 스크린샷
![panda-sprint2 netlify app_](https://github.com/user-attachments/assets/cb353e45-53c3-498b-9428-b0626e467d28)
---
![panda-sprint2 netlify app_login_](https://github.com/user-attachments/assets/a2d401a7-d5fa-4a46-be6f-b0e48ab78dfd)
---
![panda-sprint2 netlify app_sign_up_ (1)](https://github.com/user-attachments/assets/e639dd56-81cb-4139-b95c-c2d7cb8baf09)

## 멘토에게

- 보내주신 자료를 보며 위계가 잘 드러나는 네이밍을 사용하려고 했는데,
   아직은 직접 지으려고 하면 어려운 것 같습니다. 계속 연습해 보겠습니다.

- 셀프 코드 리뷰를 통해 질문 이어가겠습니다.

Commit:
add : login,sign-up

- 기존 index.html, style.css 수정
1) 헤더 상단에 고정 (style.css)
2) 외부링크 이동 시 noopener, noreferer속성 추가(index.html)
3) 설명이 필요한 interactive요소에 aria-label 속성 추가(index.html)
4) <html lang=“ko”>로 설정(index.html)

626-ju committed:
add : global.css

- toggle-view-password -> toggle-visibility-pw로 id이름 변경
- toggle-view-password-chek -> toggle-visibility-pwcheck로 id이름 변경
- global.css추가
- 포맷팅 수정oggle-view-password -> toggle-visibility-pw로 id이름 변경

626-ju committed:
add : 팔레트 변수 추가

- global.css에 시안에 있는 컬러들 추가하기
- 충돌나던 --secondary-color 변수이름 변경
- input에 placeholder추가

626-ju committed:
placeholder 수정

626-ju committed:
버튼 폰트사이즈 수정

626-ju committed:
제출준비 수정

626-ju committed:
파일 이름 수정

626-ju committed:
파일 경로 수정

addiescode-sj commented on login/index.html:
```diff
@@ -0,0 +1,47 @@
+  </head>
+  <body>
+    <main>
+      <div class="main__login">
```
main 아래에서 container 역할을 하는 요소는 `<section>` 시맨틱 태그를 사용해볼까요?

addiescode-sj commented on login/index.html:
```diff
@@ -0,0 +1,47 @@
+  <body>
+    <main>
+      <div class="main__login">
+        <h1 class="login__logo">
```
로고(CTA) 인데 태그를 heading으로 쓰셨네요! :) 엘리먼트의 역할에 적절한 태그를 사용해볼까요?

addiescode-sj commented on login/index.html:
```diff
@@ -0,0 +1,47 @@
+            <label for="user-password">비밀번호</label>
+            <input id="user-password" type="password" name="user-password"/>
+            <input id="toggle-visibility-pw" type="checkbox"/>
+            <label for="toggle-visibility-pw"></label>
```
굳굳! 이런 시도 좋아요 👍 
 여기서 아래와 같이 ARIA 속성을 추가해 접근성을 더 향상시켜볼까요?
 `aria-label="비밀번호 표시"`,`aria-checked="false"`

aria-label로 해당 요소의 목적을 설명해주고, aria-checked로 체크박스의 현재 상태(true/false)를 나타내주면 스크린 리더 사용자들에게 큰 도움이 되겠죠? :)

addiescode-sj commented on login/login.css:
```diff
@@ -0,0 +1,123 @@
+
+main {
+  font-family: 'Pretendard-Regular', sans-serif;
+}
```
혹시 페이지마다 쓰이는 폰트가 다른가요? 다르지않다면 `font-family` 또한 부모에서 자식으로 상속되니까, 여러번 쓸 필요없이 최상위 부모인 body에 default 폰트를 적용하시는게 좋을것같아요 :)

```css
body {
  font-family: "Pretendard-Regular", sans-serif;
}
```

addiescode-sj commented on login/login.css:
```diff
@@ -0,0 +1,123 @@
+input {
+  margin-top: 16px;
+  margin-bottom: 24px;
+  border: none;
```
input, button등 기본 적용되어있는 스타일을 초기화하고싶으면 매번 코드를 작성하는것보다는 초기화를 담당하는 reset.css 파일을 따로 만들어서 필요할때마다 재사용해주는게 좋겠죠?

addiescode-sj commented on sign_up/sign-up.css:
login.css와 중복되는 내용이 꽤 많은데, login.css를 해당 파일에서 import하고 sign-up에서만 쓰는 추가 스타일만 정의해보는건 어떨까요?

addiescode-sj reviewed (approved):
수고하셨습니다! 

## 주요 리뷰 포인트
- 엘리먼트 역할에 맞는 태그 사용
- 접근성 향상
- 폰트 패밀리 적용 방식
- 코드 중복 제거

626-ju committed:
refactor(mentor): sprint2 피드백

1) main 안에 section 태그 넣기
2) 이미지만 있는 로고는 헤딩 태그 쓰지 않기
3) #toggle-visiblity-pw에 aria-label="비밀번호 표시 여부",
   aria-checked="false"추가
4) 반복되는 폰트 패밀리 global.css body안에 선언
5) input 및 button 태그에 반복되는 boder:none; reset.css로 이동
6) sign-up.css에 login.css import해서 반복되는 부분 날리기

626-ju committed:
refactor: reset.css에 font-familly:inherit; cursor:pointer도 추가
